test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -925,4 +928,28 @@ public static void generateClassFile(String name, byte[] bytes) {
 		byte[] bytes = generateClass(name, fields, false, true);
 		return generator.defineClass(name, bytes, 0, bytes.length);
 	}
-}
\ No newline at end of file
+
+	public static void addgenerate2DMultiANewArray(ClassWriter cw) {
+        MethodVisitor mv = cw.visitMethod(ACC_PUBLIC + ACC_STATIC, "generate2DMultiANewArray", "(II)Ljava/lang/Object;", null, null);
+		mv.visitCode();
+		mv.visitVarInsn(ILOAD, 0);
+		mv.visitVarInsn(ILOAD, 1);
+        mv.visitMultiANewArrayInsn( "[[" + getSigFromSimpleName("Point2D", false), 2);
```

Comment:
@tajila The spot causing the error when trying to create the method

Reply:
dont hard code the name of the class, pass in the name as a param instead